### PR TITLE
[FIX] [8.0]  l10n_ch_payment_slip : line_ids is a recordset

### PR DESCRIPTION
**Pull request by eLBati**
Otherwise, importing V11 file:
`ProgrammingError: can't adapt type 'account.move.line'`

### CODE REVIEW

File: l10n_ch_payment_slip/wizard/bvr_import.py [hunk 1]
```diff
@@ -206,7 +206,7 @@ def _prepare_line_vals(self, statement, record):
         )
         if line_ids:
```
Comment (yvaucher) [inline l10n_ch_payment_slip/wizard/bvr_import.py]: In fact `line_ids` is not named properly as it is a record set.
Plus can we have multiple lines returned by search?
If yes and we still want to manage only one, we can add `limit=1` on search

Please add `limit=1` and rename `line_ids` to `line` and remove `line = line_ids`

**Comment by yvaucher** [comment]
@eLBati Thanks for the patch. See my in-line comment for some little improvements.

**Comment by eLBati** [comment]
@yvaucher I think we can't have multiple lines returned by search (@nbessi is it true?). So I added the check

File: l10n_ch_payment_slip/wizard/bvr_import.py [hunk 2]
```diff
@@ -197,16 +197,18 @@ def _prepare_line_vals(self, statement, record):
...
             [('transaction_ref', '=', reference),
              ('reconcile_id', '=', False),
              ('account_id.type', 'in', ['receivable', 'payable']),
              ('journal_id.type', '=', 'sale')],
             order='date desc',
         )
-        if line_ids:
+        if len(line) > 1:
+            raise exceptions.Warning(
+                "Too many credit lines for reference %s" % reference)
```
Comment (nbessi) [inline l10n_ch_payment_slip/wizard/bvr_import.py]: We should replace credit with receivable/payable in order not to get confused with the credit line of credit control. The exception message should be translatable.

**Comment by nbessi** [comment]
@eLBati from a business point of  view this case may happens with multi payment term invoices. The proposed message is better than the one raised by ensure_one. 

So :+1:  for the fix once the word `credit` is replaced and the translation file updated

**Comment by yvaucher** [comment]
:+1:

**Comment by eLBati** [comment]
@nbessi thanks, fixed.

> from a business point of view this case may happens with multi payment term invoices

In this case, how could the move.line be identified?

**Comment by nbessi** [comment]
@eLBati you need to use other addons like the one provided in the bank_statement_reconcile project

**Comment by mdietrichc2c** [comment]
:+1: for this fix.

**Pull request merged**